client/components/PubEdge/pubEdge.scss

```diff
@@ -0,0 +1,78 @@
+@import 'styles/variables.scss';
+@import 'containers/Pub/PubDocument/pubBody.scss';
+
+.pub-edge-component {
```

Comment:
I know this isn't explicitly called out in the designs, but since these are displayed prominently for readers, we do need to have mobile-friendly styles for them down to 320px viewports.

![image](https://user-images.githubusercontent.com/2208769/86398654-f7085200-bc73-11ea-9da5-25bba0500813.png)

I think the changes needed to the CSS here are fairly cosmetic — changing some `flex-direction` values and font sizes will get you pretty far. Positioning the image will be a little tricky. You might consider doing the image/text break vertically and using a wide, thin strip of image from `pub.headerBackgroundImage` instead of `pub.avatar` as these are typically landscape images.

You are welcome to exercise as much design freedom as you want for a mobile view, or to lean on Deepak — up to you. I'm also happy to pair further on this (or just keep armchair quarterbacking the mobile layout)